src/main/webapp/app/shared/layout/header/header.tsx

```diff
@@ -32,17 +34,20 @@ const Header = (props: IHeaderProps) => {
 
   return (
     <div id="app-header">
-      {renderDevRibbon()}
+      {/*renderDevRibbon()*/}
       <LoadingBar className="loading-bar" />
       <Navbar data-cy="navbar" dark expand="md" fixed="top" className="bg-dark">
         <NavbarToggler aria-label="Menu" onClick={toggleMenu} />
         <Brand />
         <Collapse isOpen={menuOpen} navbar>
           <Nav id="header-tabs" className="ms-auto" navbar>
             <Home />
-            {props.isAuthenticated && props.isAdmin && (
+            {/*props.isAuthenticated && props.isAdmin && (
               <AdminMenu showOpenAPI={props.isOpenAPIEnabled} showDatabase={!props.isInProduction} />
-            )}
+            )*/}
+
+            <SdkMenu />
+            <RestApiMenu />
             <AccountMenu isAuthenticated={props.isAuthenticated} />
```

Comment:
This change adds these two dropdown menus to the nav bar:

```
<SdkMenu />
<RestApiMenu />

```

These components are defined in `restapi.tsx` and `tssdk.tsx`